web/templates/stream_settings/add_subscribers_form.hbs

```diff
@@ -7,9 +7,14 @@
     </div>
     {{#if (not hide_add_button)}}
         <div class="add_subscriber_button_wrapper inline-block">
-            <button type="submit" name="add_subscriber" class="button add-subscriber-button add-users-button small rounded sea-green" tabindex="0">
-                {{t 'Add' }}
-            </button>
+            {{> ../components/action_button
+              label=(t "Add")
+              custom_classes="add-subscriber-button add-users-button"
+              id="add_subscriber"
```

Comment:
I don't think this should have an ID; the previous one did not. It was `name` that was previously `add_subscriber`

Reply:
I think this line can just be dropped, done.

Reply:
Needs this too, but that's a readability improvement:
``` diff
diff --git a/web/src/stream_ui_updates.ts b/web/src/stream_ui_updates.ts
index c46151ed60..59c5429aa2 100644
--- a/web/src/stream_ui_updates.ts
+++ b/web/src/stream_ui_updates.ts
@@ -510,7 +510,7 @@ export function enable_or_disable_add_subscribers_elements(
                 .addClass("add_subscribers_disabled");
         }
     } else {
-        const $add_subscribers_button = $container_elem.find("#add_subscriber").expectOne();
+        const $add_subscribers_button = $container_elem.find(".add-subscriber-button").expectOne();
         $add_subscribers_button.prop("disabled", !enable_elem);
         if (enable_elem) {
             $add_subscribers_button.css("pointer-events", "");
```